README.md

```diff
@@ -56,47 +56,33 @@ For production, you may want to create and fill the database using a PostgreSQL
 git clone https://github.com/datarttu/sujuikoDB
 cd sujuikoDB
 cp .example_env .env
+cp .example_env .env_test
 ```
 
-Now configure the values in .env according to your environment.
+Now configure the values in `.env` and `.env_test` according to your environment.
+Also check that the `docker-compose` files suit your needs.
 
-[`example_data`](./example_data/) is mapped as import data directory by default.
-If you are not just testing with the example data, set `IMPORT_DATA_DIR` in the `.env` file to a directory that contains the data you want to import when starting the db instance.
+## Testing the DDL and data imports with example data
 
-[`db`](./db/) is mapped to the container as well, as `/db` directory, so any SQL scripts in it can be run inside the container as server-side processes.
+1. In `.env_test`, set `IMPORT_DATA_DIR` to `./example_data`.
+1. Start the *test run* version of the database: `docker-compose -f docker-compose.test.yml up`.
```

Comment:
We could add `docker build -t datarttu/sujuiko:latest .` before running `docker-compose`. `docker-compose ... --build` is not always reliable, I have been told.

Reply:
👍 #40